src/lambda/commands/deploySamApplication.ts

```diff
@@ -100,18 +100,19 @@ export async function deploySamApplication({
             // Deploying can take a very long time for Python Lambda's with native dependencies so user needs feedback
             restParams.outputChannel.appendLine(localize(
                 'AWS.samcli.deploy.workflow.stackName.initiated',
-                'Deploying SAM Application to CloudFormation Stack: {0}',
-                stackName
+                'Deploying SAM Application to CloudFormation Stack: {0} with profile: {1}',
+                stackName, profile
             ))
             await deployInvocation.execute()
 
-            msg = localize(
+            const msg = localize(
                 'AWS.samcli.deploy.workflow.success',
                 'Successfully deployed SAM Application to CloudFormation Stack: {0} with profile: {1}',
                 stackName, profile
             )
+            outputFinalMessage(msg, restParams.outputChannel)
```

Comment:
`outputFinalMessage` is showing Warning, this should be showing Information

Reply:
✔️ , d'oh